packages/components/divider/src/divider.ts

```diff
@@ -13,5 +13,9 @@ export const dividerProps = buildProps({
     values: ['left', 'center', 'right'],
     default: 'center',
   },
+  lineStyle: {
```

Comment:
add values like `contentPosition` ['left', 'center', 'right']?

Reply:
now, users can set lots of style like border-style, 
![image](https://user-images.githubusercontent.com/21104054/144421378-b37d37b2-20eb-4a72-9105-621d082da03e.png)
so i delete the values.
do you think we should add them all ,or just add ['solid','doshed','dotted','double'](this will limit user)?

Reply:
emm, if we can use type to define props, we can use `CSSStyleDeclaration['borderStyle']` directly.

@sxzz Is there any better way?

Reply:
Yeah, we can use `definePropType`.
Here is pseudo code
```
type: definePropType<BorderStyle>(String)
```